Euphorolog.Database/ModelConfigurations/UsersConfiguration.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Euphorolog.Database.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Euphorolog.Database.ModelConfigurations
+{
+    public class UsersConfiguration : IEntityTypeConfiguration<Users>
+    {
+        public void Configure(EntityTypeBuilder<Users> builder)
+        {
+            //userName
+            builder
+                .HasKey("userName");
+            builder
+                .Property(u => u.userName)
```

Comment:
When you define something as a primary key, you do not need to call it `Required` or `Unique` explicitly.